### PR TITLE
Rename _token to token

### PR DESCRIPTION
**Pull request by kpman**

### CODE REVIEW

File: src/cli/providers/messenger/whitelisted-domains.js [hunk 1]
```diff
@@ -34,13 +34,13 @@ const help = () => {
 };
 
 export async function getWhitelistedDomains(ctx) {
-  const { t, token: _token } = ctx.argv;
+  const { t, token } = ctx.argv;
```
Comment (chentsulin) [inline src/cli/providers/messenger/whitelisted-domains.js]: How about 

```js
const token = ctx.argv.token || ctx.argv.t;
```

File: src/cli/providers/messenger/attachment.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ const logUploadInfo = uploadInfo => {
 };
 
 export async function uploadAttachment(ctx) {
-  const { f, force: _force, y, yes: _yes, t, token: _token } = ctx.argv;
+  const { f, force: _force, y, yes: _yes, t, token } = ctx.argv;
```
Comment (chentsulin) [inline src/cli/providers/messenger/attachment.js]: We should rename `_force`, `_yes` too.

File: src/cli/providers/messenger/whitelisted-domains.js (hunk 1 above)
Comment (chentsulin) [reply src/cli/providers/messenger/whitelisted-domains.js]: ping @kpman

File: src/cli/providers/messenger/attachment.js (hunk 2 above)
Comment (kpman) [reply src/cli/providers/messenger/attachment.js]: We use `force` & `yes` as variable for ctx.argv boolean result.

```js
const force = f || _force;
const yes = y || _yes;
```

That's why we need to rename `force` as `_force`, `yes` as `_yes`.
And we use `accessToken` for the `token` & `t` boolean result, so we don't need to rename `token`.

Comment (kpman) [reply src/cli/providers/messenger/attachment.js]: or maybe change all codebase to the following convention:

```js
const force = ctx.argv.f || ctx.argv.force;
const yes = ctx.argv.y || ctx.argv.yes;
const token = ctx.argv.t || ctx.argv.token;

const accessToken = token;
```

@chentsulin what do you think?

File: src/cli/providers/messenger/whitelisted-domains.js (hunk 1 above)
Comment (kpman) [reply src/cli/providers/messenger/whitelisted-domains.js]: Inspired by this idea, I left some comments here https://github.com/Yoctol/bottender/pull/349#discussion_r228774753

**Pull request closed without merging**